### PR TITLE
ISO layout for KBD67 rev 2.0

### PR DESCRIPTION
**Pull request by hekekee**
## Description

KBD67 rev 2.0 was missing a ISO layout keymap, made with `LAYOUT_65_iso`.
This provides easier starting point for future keymap customization for ISO layout users.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [X] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [X] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [X] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [X] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/kbdfans/kbd67/rev2/keymaps/iso/keymap.c [hunk 1]
```diff
@@ -0,0 +1,64 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/kbdfans/kbd67/rev2/keymaps/iso/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/kbdfans/kbd67/rev2/keymaps/iso/config.h [hunk 2]
```diff
@@ -0,0 +1,19 @@
+/* Copyright 2018 MechMerlin
```
Comment (drashna) [inline keyboards/kbdfans/kbd67/rev2/keymaps/iso/config.h:1]: Can be removed. Is no override to be placed.

File: keyboards/kbdfans/kbd67/rev2/keymaps/iso/keymap.c [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+}
+
+void matrix_scan_user(void) {
+
+}
+
+void led_set_user(uint8_t usb_led) {
+
+}
```
Comment (drashna) [inline keyboards/kbdfans/kbd67/rev2/keymaps/iso/keymap.c:64]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by hekekee** [comment]
Made the changes according to @drashna's comments. This is my first ever pull request on Github, thanks for being gentle ♥

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**